### DISCUSSION TITLE
[windows] Is there any plans for developing this feature: change the installation path

### DISCUSSION THREAD

**Discussion by gitikid** [toeverything/AFFiNE#5850] [Q&A] [answered]
Hi, I like your product. I would like to inquire if there are any plans to support this feature: when installing this software, choose the installation path and the local note storage folder（when using windows）

**Comment by HeJiachen-PM** [comment, marked as answer]
Sure thing! We will implement it once we have a stable installer

**Reply by gitikid** [reply]
Thx! expect it so much
